ui/src/vayla-design-lib/closeable-modal/closeable-modal.tsx

```diff
@@ -46,8 +51,12 @@ export const CloseableModal: React.FC<CloseableModalProps> = ({
     useResizeObserver({
         ref: document.body,
         onResize: () => {
-            setX(positionRef.current?.getBoundingClientRect().x);
-            setY(positionRef.current?.getBoundingClientRect().y);
+            setX(boundingRect?.x);
+            setY(boundingRect?.y);
```

Comment:
En ole ihan varma, oliko alkuperäinenkään toiminnallisuus kuinka hyvin synkassa ikkunan koonmuutosten kanssa, mutta tämän kanssa ainakin voi useResizeObserver()in callbackiin jäädä vanhoja boundingRectin arvoja, jolloin modaalin sijainti jää päivittymättä. Ikkunan raameja raahaamalla voi olla vaikea todentaa, mutta ainakin minulla onnistuu helposti napsuttelemalla Firefoxia koko ruudun vs. ikkunan kokoiseksi.